scenarioo-server/src/main/java/org/scenarioo/rest/application/ScenariooWebApplication.java

```diff
@@ -35,41 +35,42 @@
  * Scenarioo REST Services Web Application context that initializes data on startup of server.
  */
 public class ScenariooWebApplication implements ServletContextListener {
-	
+
```

Comment:
I think we have to discuss about our formattings in the code:
1. Each developer should in general make sure to not commit such changes on a feature branch and talk to us to find a better solution
2. Option A: we adjust the formatting of the whole code to better conform to IntelliJ Java default settings and document how to configure same settings in Eclipse
3. Option B: we find a way to configure also IntelliJ to use the same formattings and force every developer to use those.

I vote for option A - let's make our style of code formatting more "common sense" - no extra rules, that most developers do not have in their settings.

Nevertheless, developers should avoid to commit such changes on a feature branch or fork